build.roc

```diff
@@ -0,0 +1,153 @@
+app [main] {
+    cli: platform "platform/main.roc",
```

Comment:
Need to replace this with the pre-release when we have it available. 

You can use `jump-start.sh` to build the platform from scratch while this is `platform/main.roc` -- for e.g. building said pre-release.